<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsynchronousOneWaveformAOClass</class>
 <widget class="QDialog" name="AsynchronousOneWaveformAOClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>339</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>339</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Asynchronous One Waveform AO</string>
  </property>
  <widget class="QFrame" name="background">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>-3</y>
     <width>340</width>
     <height>431</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>340</width>
     <height>431</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#background{background-image:url(:/AsynchronousOneWaveformAO/Resources/ao.png)}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="BtnSineA">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>127</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/AsynchronousOneWaveformAO/Resources/sine.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnSquareA">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>127</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/AsynchronousOneWaveformAO/Resources/square.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnTriangleA">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>127</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:AsynchronousOneWaveformAO/Resources/triangle.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnSineB">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>310</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/AsynchronousOneWaveformAO/Resources/sine.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnTriangleB">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/AsynchronousOneWaveformAO/Resources/triangle.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnSquareB">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>310</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/AsynchronousOneWaveformAO/Resources/square.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtboxHiLevelA">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>91</y>
      <width>73</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>73</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>73</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtboxLoLevelA">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>90</y>
      <width>73</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>73</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>73</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>-5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtboxLoLevelB">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>275</y>
      <width>73</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>73</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>73</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>-5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtboxHiLevelB">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>276</y>
      <width>73</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>73</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>73</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="chLabelA">
    <property name="geometry">
     <rect>
      <x>228</x>
      <y>33</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 170, 0)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="chLabelB">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>218</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 170, 0)</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfigure">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>394</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>75</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string>Configure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>394</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>75</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
